pages/settingspage.py

```diff
@@ -106,32 +123,29 @@ def set_data(self):
             self.buy_one_checkbox.setChecked(True)
         if settings['dont_buy']:
             self.dont_buy_checkbox.setChecked(True)
+        self.random_delay_start.setText(settings["random_delay_start"])
+        self.random_delay_stop.setText(settings["random_delay_stop"])
         self.update_settings(settings)
 
     def save_settings(self):
-        settings = {"webhook":self.webhook_edit.text(),
-                    "webhookonbrowser":self.browser_checkbox.isChecked(),
-                    "webhookonorder":self.order_checkbox.isChecked(),
-                    "webhookonfailed":self.paymentfailed_checkbox.isChecked(),
-                    "browseronfailed":self.onfailed_checkbox.isChecked(),
-                    "onlybuyone":self.buy_one_checkbox.isChecked(),
-                    "dont_buy":self.dont_buy_checkbox.isChecked()}
+        settings = {"webhook":            self.webhook_edit.text(),
+                    "webhookonbrowser":   self.browser_checkbox.isChecked(),
+                    "webhookonorder":     self.order_checkbox.isChecked(),
+                    "webhookonfailed":    self.paymentfailed_checkbox.isChecked(),
+                    "browseronfailed":    self.onfailed_checkbox.isChecked(),
+                    "onlybuyone":         self.buy_one_checkbox.isChecked(),
+                    "dont_buy":           self.dont_buy_checkbox.isChecked(),
+                    "random_delay_start": self.random_delay_start.text(),
+                    "random_delay_stop":  self.random_delay_stop.text()}
         write_data("./data/settings.json",settings)
         self.update_settings(settings)
         QtWidgets.QMessageBox.information(self, "Phoenix Bot", "Saved Settings")
 
-    def update_settings(self,settings_data):
-        global webhook, webhook_on_browser, webhook_on_order, webhook_on_failed, browser_on_failed, dont_buy
+    def update_settings(self, settings_data):
+        global webhook, webhook_on_browser, webhook_on_order, webhook_on_failed, browser_on_failed, dont_buy, rand_delay_start, rand_delay_stop
```

Comment:
Is this misspelled? I think it should be `random_delay_start ` & `random_delay_stop`

Reply:
In settings.py I spelled them rand_delay_start, rand_delay_stop so it is not a bug.

Reply:
yeah i think that's what initially threw me off. Lets make it consistent then.

Reply:
Fixed!